### PR TITLE
new icon: veevalidate (original, line)

### LINKED ISSUE #1834: [ICON REQUEST]: vee-validate
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

VeeValidate is a library that takes care of value tracking, validation, errors, submissions and more

### Links and sources

Check out the [vee-validate website](https://vee-validate.logaretm.com/v4/)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by canaleal**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1834**

## Link to prove your SVG is correct and up-to-date.

- [VeeValidate Website](https://vee-validate.logaretm.com/v4/)
- [VeeValidate Github](https://github.com/logaretm/vee-validate/)
- [VeeValidate NPM Docs](https://www.npmjs.com/package/vee-validate)

It seems that VeeValidate lacks an official Wordmark. While the website showcases the VeeValidate logo next to some text, this text is encapsulated within a span tag and is not integrated into the official logo itself. 

For the original SVG, I kept the official green `#04c971` and unified the all paths to make use of aliases (Plain).

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -10103,6 +10103,30 @@
...
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#05c770",
```
Comment (Snailedlt) [inline devicon.json:10122]: This color is different from the one used in the svg
```suggestion
        "color": "#04c971",
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]

File: devicon.json (hunk 1 above)
Comment (canaleal) [reply devicon.json]: Apologies, my mistake. Color has been updated.

Comment (Snailedlt) [reply devicon.json]: No worries! 😁

**Review by Snailedlt — DISMISSED** [review]
LGTM! ✔️ 
Thank you for another great contribution! 💯 🙇
Excellent work!

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️
Looks Great! 💯 Thank you for another great PR! 🙇‍♂️
image
image
image

**Review by weh — APPROVED** [review]
LGTM :+1:

**Pull request merged**